docs/mivot/example.rst

```diff
@@ -138,58 +136,65 @@ which models a full source's  astrometry at a given date.
 
 
 .. warning::
-   At the time of writing, Vizier only mapped positions and proper motions (when  available),
+   At the time of writing (Q1 2025), Vizier only mapped positions and proper motions (when  available),
    and the definitive epoch class had not been adopted.
    Therefore, this implementation may differ a little bit from the standard model.
 
    Vizier does not wrap the source properties in a MANGO object,
    but rather lists them in the Mivot *TEMPLATES*.
-   The annotation reader must support both designs.
+   The annotation reader supports both designs.
 
 In the first step below, we run a standard cone search query by using the standard PyVO API.
-
- .. code-block:: python
-
-    import pytest
-    import astropy.units as u
-    from astropy.coordinates import SkyCoord
-    from pyvo.dal.scs import SCSService
-
-    from pyvo.utils import activate_features
-    from pyvo.mivot.viewer.mivot_viewer import MivotViewer
-    from pyvo.mivot.features.sky_coord_builder import SkyCoordBuilder
-    from pyvo.mivot.utils.dict_utils import DictUtils
-
-    # Enable MIVOT-specific features in the pyvo library
-    activate_features("MIVOT")
-
-    scs_srv = SCSService("https://vizier.cds.unistra.fr/viz-bin/conesearch/V1.5/I/239/hip_main")
-
-    query_result = scs_srv.search(
-        pos=SkyCoord(ra=52.26708 * u.degree, dec=59.94027 * u.degree, frame='icrs'),
-        radius=0.5)
-
-    # The MIVOt viewer generates the model view of the data
-    m_viewer = MivotViewer(query_result, resolve_ref=True)
-
 Once the query is finished, we can get a reference to the object that will process the Mivot annotations.
 
- .. code-block:: python
-
-    # Build a Python object matching the TEMPLATES content and
-    # which leaves are set with the values of the first row
-    mango_property = m_viewer.dm_instance
-
-    # Print out the content of the Python object
-    # This statement is just for a pedagogic purpose
-    DictUtils.print_pretty_json(mango_property.to_dict())
-
-The annotations are consumed by this dynamic Python object which leaves are set with the data of the current row.
-You can explore the structure of this object by using standard object paths or by browsing the dictionary shown below.
-
- .. code-block:: json
-
-	{
+.. doctest-skip::
+
+   >>> import pytest
+   >>> import astropy.units as u
+   >>> from astropy.coordinates import SkyCoord
+   >>> from pyvo.dal.scs import SCSService
+   >>> from pyvo.utils import activate_features
+   >>> from pyvo.mivot.viewer.mivot_viewer import MivotViewer
+   >>> from pyvo.mivot.features.sky_coord_builder import SkyCoordBuilder
```

Comment:
This is not an issue at all, but I would think that in narrative documentation there is a benefit of doing the imports at piecemeal -- only in the cells where they are (first) used, where the classes/functions are being first described as opposed to doing everything at the top.